core/iwasm/common/wasm_runtime_common.c

```diff
@@ -4212,6 +4212,11 @@ static V128FuncPtr invokeNative_V128 = (V128FuncPtr)(uintptr_t)invokeNative;
           || defined(BUILD_TARGET_RISCV64_LP64) */
 #endif /* end of defined(_WIN32) || defined(_WIN32_) */
 
+#if defined(__GNUC__)
+#pragma GCC push_options
+#pragma GCC optimize("O0")
+__attribute__((no_sanitize_address)) 
```

Comment:
Why need to disable the address sanitizer? Are there any operations which cannot meet the requirements?

Reply:
Looks like we may have a couple of false-positive `ASAN` errors. 

I'm just trying to find what's the error here (or probably no error as we use custom stack unwind as I got from the code, which might cause ASAN false-positive itself)

Reply:
if you decide to keep it, it'd be worth adding a comment why we need it - next time somebody refactors the code might be able to remove it.